src/device-registry/utils/create-event.js

```diff
@@ -493,12 +460,28 @@ const createEvent = {
       let page = parseInt(query.page);
       const language = request.query.language;
       const filter = generateFilter.events(request);
-      const cacheResult = await createEvent.getCache(request);
-      logObject("Cache result", cacheResult);
 
-      if (cacheResult.success === true) {
-        logText(cacheResult.message);
-        return cacheResult.data;
+      try {
+        const cacheResult = await Promise.race([
+          createEvent.getCache(request),
```

Comment:
properly handle cases where the Redis server is off, just to ensure that the entire function does not hang. @BenjaminSsempala